ui/src/virtualconsole/vcxypadfixture.cpp

```diff
@@ -557,7 +557,9 @@ void VCXYPadFixture::readDMX(QList<Universe*> universes, qreal & xmul, qreal & y
         return;
 
     if (m_xReverse == true)
```

Comment:
1- the language to be used is english
2- adding those debug lines breaks the if condition
3- please remove those lines, as writeDMX is a high frequency called method